content/automate/ManageIQ/Cloud/VM/Provisioning/StateMachines/ProvisionRequestApproval.class/__methods__/validate_request.rb

```diff
@@ -126,7 +126,8 @@ def flavor_value(prov_resource, option)
 # Use value from model unless specified above
 max_memory ||= $evm.object['max_memory']
```

Comment:
@billfitzgerald0120 - is it always the case that `$evm.object['max_memory']` comes in the same way? The reporter on the BZ made mention that Amazon does one thing ( bytes ) but what about other cloud providers?

Reply:
@syncrou Yes, ```$evm.object['max_memory']``` always comes in the same way. The provider values differ, but that doesn't affect this instance variable.